src/connect.js

```diff
@@ -20,7 +20,11 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  return newObj => {
```

Comment:
you can return object here
```suggestion
  return newObj => { ...params,  ... newObj };
```

Reply:
Ok)

Reply:
додав дужки до прикладу, без них буде помилка